custom-namespaces.yaml

```diff
@@ -1369,6 +1368,17 @@ shared:
       type: table_view
       tables:
         - table: mozdata.telemetry.applications
+shared_private:
```

Comment:
If they just need to reference the generated shared `countries` view then it doesn't seem like this `shared-private` spoke namespace is necessarily needed, as `looker-hub` views can be referenced in `looker-spoke-private`.

Reply:
That said, I guess having the spoke namespace would allow for refining the `countries` view, though in that case any refinements would have to be duplicated in both `looker-spoke-default` and `looker-spoke-private` (and there are [already refinements to the `countries` view in `looker-spoke-default`](https://github.com/mozilla/looker-spoke-default/blob/main/shared/views/countries.view.lkml)).

I wonder if it would make sense to have `looker-spoke-default` referencable from `looker-spoke-private` (similar to how `looker-hub` is)?

Reply:
I just tried. About 100 errors in validation. I guess it might not be happy about `.model` files being present in spoke-default (which aren't in looker-hub).